frontend/app/api/hal-api-resource.js

```diff
@@ -62,6 +62,22 @@ module.exports = function HALAPIResource($timeout, $q, PathHelper) {
       return new Hyperagent.Resource({
         url: PathHelper.appBasePath + PathHelper.apiV3 + '/' + uri,
       });
+    },
+    setupLink: function(uri, params) {
+      // some monkey-patching here
+      // since the Hyperagent.Link constructor
+      // behaves differently from the links constructed
+      // from the API response
+      HALAPIResource.configure();
+      var url = PathHelper.appBasePath + PathHelper.apiV3 + '/' + uri;
```

Comment:
This is looking suspicious:

A hard reference to APIv3 in a neutral looking class. Since it is called "monkey patching" I suppose that this method is called/ used outside of our application too? Because for our application I can only find a single call in the work-package-service.